superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -63,14 +64,22 @@ export function datasetReducer(
 export default function AddDataset() {
   // this is commented out for now, but can be commented in as the component
```

Comment:
we can delete this comment now that we are actually using the reducer :)

Reply:
+1 on this, remove the "this is commented out for now..." comment